### DISCUSSION TITLE
мобильная версия для Ios

### DISCUSSION THREAD

**Discussion by punisherkrox** [stoatchat/discussions#811] [Feature Suggestions]
было бы здорово мобильно приложение на андроид и ios , будет ли в дальнейшем? из-за блокировки discord  на территории РФ Ваша программа отличный аналог и для нашего комьюнити геймеров отличная платформа, но для поддержки связи нужна и мобильная версия

**Comment by mikki33** [comment]
На андроид уже есть приложение, ищите в гуглплей. Это я нашел так у них на сайте.
P.s. на сайте есть строка есть надпись "See other platforms and downloads". Если на неё кликнуть откроется список что доступно из приложений, а что нет.
